sigridci/sigridci.py

```diff
@@ -608,15 +613,15 @@ def displayMetadata(self, apiClient):
         print("You cannot use both --publish and --pathprefix at the same time, refer to the documentation for details")
         sys.exit(1)
 
-    if not SYSTEM_NAME_PATTERN.match(args.system):
-        print("Invalid system name, system name can only contain letters/numbers/hyphens, and cannot start with a hyphen")
-        sys.exit(1)
-
     log("Starting Sigrid CI")
+    
     options = UploadOptions(args.source, args.exclude.split(","), args.include_history, args.pathprefix, args.showupload, args.publishonly)
     target = TargetQuality(options.readScopeFile() or "", args.targetquality)
     apiClient = SigridApiClient(args)
     reports = [TextReport(), StaticHtmlReport(), JUnitFormatReport(), ExitCodeReport()]
 
     runner = SigridCiRunner()
+    if not runner.isValidSystemName(args.system):
+        print(f"Invalid system name, system name should match {SYSTEM_NAME_PATTERN} and within {SYSTEM_NAME_LENGTH} characters")
```

Comment:
Gramatically I think you should add "be" between "and" and "within", however if I'm not mistaken `{SYSTEM_NAME_LENGTH}` is printed as "range(2, 65)" which doesn't read very nicely. Also directly using `{SYSTEM_NAME_PATTERN}` gives "re.compile("^[a-z0-9]+(-[a-z0-9]+)*$", re.IGNORECASE)" and I think it's nicer to stick to just the pattern. I'd go for something like:

```suggestion
        print(f"Invalid system name, system name should match '{SYSTEM_NAME_PATTERN.pattern}' "
              f"and be {SYSTEM_NAME_LENGTH.start} to {SYSTEM_NAME_LENGTH.stop} characters long (inclusive).")
```

Reply:
Whoops, you're totally right. Those fields used to be a string and integer originally, but that way of printing indeed isn't very friendly.